Sources/SwiftParser/Expressions.swift

```diff
@@ -2110,8 +2134,22 @@ extension Parser {
     } else {
       conditions = self.parseConditionList()
     }
-
-    let body = self.parseCodeBlock(introducer: ifKeyword)
+    
+    
+    var body: RawCodeBlockSyntax
+    
+    if experimentalFeatures.contains(.trailingComma) {
+      if !self.at(prefix: "{"), let blockAfterTrailingComma = extractCodeBlockFromLast(&conditions) {
```

Comment:
Should be able to simplify this slightly.

```suggestion
      if !self.at(.leftBrace), let blockAfterTrailingComma = extractCodeBlockFromLast(&conditions) {
```